package.json

```diff
@@ -10,10 +10,12 @@
     "@types/node": "^16.18.68",
     "@types/react": "^18.2.45",
     "@types/react-dom": "^18.2.18",
+    "@types/styled-components": "^5.1.34",
```

Comment:
I don't think we should start using styled components now since we already started using scss modules